### DISCUSSION TITLE
Capacitor : android Listen to share event

### DISCUSSION THREAD

**Discussion by jamalroger** [ionic-team/capacitor#3383] [Q&A] [answered]
hi developpers ,i am trying use share plugin (https://capacitorjs.com/docs/apis/share) to listen to share event to get share information from other app 
i  added my app to list menu share by add this code to manifest 
```xml
 <intent-filter>
        <action android:name="android.intent.action.SEND" />
        <category android:name="android.intent.category.DEFAULT" />
        <data android:mimeType="text/plain" />
    </intent-filter>
```
the question is how to get info share to my app 
i used something like this but not work
```js
    Share.addListener("share", (event) => {
      Plugins.Modals.alert({
        title: "share event",
        message: "it works",
      });
   });
```

**Comment by jcesarmobile** [comment, marked as answer]
There is no share event, capacitor share plugin only shares to other apps, but doesn’t receive. You’ll have to build your own plugin for that.

**Comment by jamalroger** [comment]
i find this plugin https://github.com/carsten-klaffke/send-intent 
add it to capacitor community plugins it's helpfull

**Reply by imhoffd** [reply]
We're discussing it here: https://github.com/capacitor-community/proposals/issues/46
